packages/cli/src/commands/setup/tsconfig/tsconfig.js

```diff
@@ -3,6 +3,8 @@ import path from 'path'
 import chalk from 'chalk'
 import Listr from 'listr'
 
+import { errorTelemetry } from '@redwoodjs/internal'
```

Comment:
😢

Reply:
hmmm... something isn't right, I wonder if a merge went bad. I definitely updated these imports... I can see missing one, but I definitely didn't miss all of them...

Reply:
yeah I'm sure I checked the other ones too... but all these new ones appeared :S

Reply:
I just merged in `main` but there weren't any conflicts...hmm. Fixed them all manually.